vnext/Scripts/OfficeReact.Win32.nuspec

```diff
@@ -35,7 +35,7 @@
 
     <file src="$nugetroot$\inc\Shared\AbiSafe.h" target="inc"/>
     <file src="$nugetroot$\inc\Shared\DevSettings.h" target="inc"/>
-    <file src="$nugetroot$\inc\Desktop\INativeUIManagerLegacy.h" target="inc"/>
+    <file src="$nugetroot$\inc\ReactWin32\INativeUIManagerLegacy.h" target="inc"/>
```

Comment:
Wat's wrong with "Desktop"?

Also, we could perhaps use the `AppPlatform` naming convention instead?
https://docs.microsoft.com/en-us/windows/apps/desktop/choose-your-platform#win32